src/library/scala/sys/package.scala

```diff
@@ -61,7 +61,10 @@ package object sys {
    *
    *  @return   a Map containing the system environment variables.
    */
-  def env: immutable.Map[String, String] = immutable.Map.from(System.getenv().asScala)
+  def env: immutable.Map[String, String] = {
+    immutable.Map.from(System.getenv().asScala)
+      .withDefault(System.getenv)
```

Comment:
Should throw on null?

Reply:
i.e., IIUC, `Map.apply` throws on no element, or `withDefault` which we suggest here should also throw on no element instead of returning null. These are the semantics of `Map.apply`.